fineract-provider/src/main/java/org/apache/fineract/batch/command/internal/GetLoanTransactionByExternalIdCommandStrategy.java

```diff
@@ -48,7 +48,7 @@
  */
 @Component
 @RequiredArgsConstructor
-public class GetTransactionByExternalIdCommandStrategy implements CommandStrategy {
+public class GetLoanTransactionByExternalIdCommandStrategy implements CommandStrategy {
 
     /**
```

Comment:
Arent these changes irrelevant to [FINERACT-1921](https://issues.apache.org/jira/browse/FINERACT-1921)? Please extract them into a separate PR if so!